pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -86,8 +103,8 @@ func TestDevicePluginReRegistration(t *testing.T) {
 	devices := m.Devices()
 	require.Equal(t, 2, len(devices[testResourceName]), "Devices are not updated.")
 
-	p2 := NewDevicePluginStub(devs, pluginSocketName+".new")
-	err := p2.Start()
+	p2 := NewDevicePluginStub(devs, pluginSocketName+".TestDevicePluginReRegistration.new")
```

Comment:
I dont think `TestDevicePluginReRegistration` is really needed.

Reply:
Right, this was an experimental change before changing the dir. Removed.